be/src/exec/pipeline/operator.cpp

```diff
@@ -10,11 +10,18 @@
 #include "util/runtime_profile.h"
 
 namespace starrocks::pipeline {
+
+const int32_t Operator::s_pseudo_plan_node_id_for_result_sink = -99;
```

Comment:
result_sink should has the real plan node id

Reply:
Hide ResultSink's plan_node_id